server/tinylicious/src/runner.ts

```diff
@@ -22,7 +22,7 @@ import detect from "detect-port";
 import * as app from "./app";
 
 export class TinyliciousRunner implements IRunner {
-    private server: IWebServer;
+    private server?: IWebServer;
```

Comment:
Can this ever be undefined?

Reply:
Yeah, the constructor doesn't assign it, and if `start()` got an exception before `server` is assigned, it will be `undefined`.

The `tinylicious` package is currently build with `strictNullCheck` as false, which would have caught the issue.  I will do a separate PR to turn it on.